Fixes:
  Error relocating /usr/lib/libmozjs-52.so: _ZN2js16TlsPerThreadDataE: symbol not found

--- mozjs-52.9.1pre1/mfbt/ThreadLocal.h.orig	2018-11-28 09:48:43.746859132 +0000
+++ mozjs-52.9.1pre1/mfbt/ThreadLocal.h	2018-11-28 09:48:45.930897382 +0000
@@ -211,7 +211,7 @@
 }
 
 #ifdef MOZ_HAS_THREAD_LOCAL
-#define MOZ_THREAD_LOCAL(TYPE) __thread mozilla::detail::ThreadLocal<TYPE>
+#define MOZ_THREAD_LOCAL(TYPE) MOZ_EXPORT __thread mozilla::detail::ThreadLocal<TYPE>
 #else
 #define MOZ_THREAD_LOCAL(TYPE) mozilla::detail::ThreadLocal<TYPE>
 #endif
